src/test/assembly/stack-protector/stack-protector-target-support.rs

```diff
@@ -0,0 +1,201 @@
+// Test that stack smash protection code is emitted for all tier1 and tier2
+// targets, with the exception of nvptx64-nvidia-cuda
```

Comment:
Based on the CI failure, this needs a `min-llvm-version` tag. For reference:

> rustc -Cllvm-args="..." with: Unknown command line argument '--x86-experimental-lvi-inline-asm-hardening'.

Though this is also going to fail for anyone locally building LLVM with anything less than the full set of targets. Availability can be tested with `needs-llvm-components`.

Reply:
Thank you, I was hoping someone would just tell me what to do with that failure :)